components/tools/OmeroPy/src/omero/plugins/admin.py

```diff
@@ -1398,13 +1398,14 @@ def port_val(port_type, value):
         from omero.plugins.web import WebControl
         try:
             WebControl().status(args)
-        except:
-            self.ctx.out("OMERO.web not installed!")
+        except Exception, e:
+            self.ctx.err("OMERO.web error:")
+            self.ctx.err(e)
```

Comment:
Discussed with @aleksandra-tarkowska, this seems to be catching an exception thrown by `self.ctx.die()` in which case a tuple is returned.